jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  moduleNameMapper: {
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|css|less)$": "<rootDir>/__mocks__/fileMock.js",
+  },
+  "setupFilesAfterEnv": ["<rootDir>/template/src/setupTests.js"]
+}
```

Comment:
Is this just for running the template's test or do we plan on adding other tests outside the template directory in the future?